elfreloc_amd64.c

```diff
@@ -26,6 +26,9 @@ el_status el_applyrela(el_ctx *ctx, Elf_RelA *rel)
     uint32_t type = ELF_R_TYPE(rel->r_info);
     uint32_t sym  = ELF_R_SYM(rel->r_info);
 
+    (void) sym;
+    (void) ctx;
```

Comment:
Thinking about this, I think I'd rather remove the name ctx from the function? definition?

i.e.

```
el_status el_applyrela(el_ctx *, Elf_Rela *rel)
```

Reply:
ctx is used in a couple of statements in this function